linchpin/hooks/action_managers/ansible_action_manager.py

```diff
@@ -162,6 +162,7 @@ def get_ctx_params(self):
         ctx_params["resource_file"] = self.target_data.get("resource_file",None)
         ctx_params["layout_file"] = self.target_data.get("layout_file",None)
         ctx_params["inventory_file"] = self.target_data.get("inventory_file",None)
+        ctx_params["inventory_dir"] = self.target_data.get("inventory_dir",None)
```

Comment:
And thinking about it, I almost prefer this suggestion, because inventory_dir is definitely *not* a useful output of linchpin, and is really an ansible-specific thing that should live only in the ansible action manager.